bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
Hmm... the downside of `getConfigForFile()` is, when checking an eslint config outside of the own project's path it will inevitably break, because the path to lookup for extended configs [will be reset to the own project's](https://github.com/eslint/eslint/blob/master/lib/config/config-file.js#L298-L318).

This would be no problem as long as `eslint-find-new-rules` is installed locally on a project's package and only used on configs that are contained within that project.
Installing `eslint-find-new-rules` globally and would **not** work, as referenced configs couldn't be found.